test/common/http/http1/codec_impl_test.cc

```diff
@@ -2120,8 +2120,9 @@ TEST_F(Http1ServerConnectionImplTest, ShouldDumpDispatchBufferWithoutAllocatingM
 class Http1ClientConnectionImplTest : public Http1CodecTestBase {
 public:
   void initialize() {
-    codec_ = std::make_unique<Http1::ClientConnectionImpl>(
-        connection_, http1CodecStats(), callbacks_, codec_settings_, max_response_headers_count_);
+    codec_ = std::make_unique<Http1::ClientConnectionImpl>(connection_, http1CodecStats(),
+                                                           callbacks_, codec_settings_,
+                                                           max_response_headers_count_, false);
```

Comment:
nit: /* passing_through_proxy=*/ false (or use a local variable `bool passing_through_proxy = false`)

Reply:
done.  though for the record I dislike that paradigm